core/src/main/java/org/mapfish/print/output/AbstractJasperReportOutputFormat.java

```diff
@@ -84,8 +86,8 @@ protected abstract void doExport(final OutputStream outputStream, final Print pr
             throws JRException, IOException;
 
     @Override
-    public final void print(final String jobId, final PJsonObject requestData, final Configuration config,
-                            final File configDir, final File taskDirectory, final OutputStream outputStream)
+    public final Processor.ExecutionContext print(final String jobId, final PJsonObject requestData, final Configuration config,
+                                                  final File configDir, final File taskDirectory, final OutputStream outputStream)
```

Comment:
Too long lines ...

Reply:
Will fix the long line in another PR. The plan is to have IDEA auto-format all the files once for good.